services/src/handlers/ebv.rs

```diff
@@ -616,6 +628,75 @@ impl<C: Context> Task<C::TaskContext> for EvbOverviewTask<C> {
 
 impl TaskStatusInfo for NetCdfCfOverviewResponse {}
 
+#[derive(Debug, Deserialize)]
+struct RemoveOverviewParams {
+    file: PathBuf,
```

Comment:
wouldn't it be better to give the dataset name or id?

Reply:
Currently, the Path is the identification… it is not ideal, but that is also used for lookups currenty…